src/screens/SendCoinsConfirmScreen.tsx

```diff
@@ -211,8 +211,7 @@ class SendCoinsConfirmScreen extends Component<Props> {
           <View>
             <View style={styles.chooseWalletButton}>
               <Text style={typography.headline4}>
-                {roundBtcToSatoshis(item.amount) || satoshiToBtc(item.value).toString()}{' '}
-                {fromWallet.preferredBalanceUnit}
+                {item.amount.toFixed(8) || satoshiToBtc(item.value).toString()} {fromWallet.preferredBalanceUnit}
```

Comment:
don't we need that ugly space between value and unit?

Reply:
true

Reply:
but the problem is my Vscode removed this 
have clue how manage that?

Reply:
done